app/src/main/java/com/example/testapp/MainActivity.kt

```diff
@@ -10,7 +10,7 @@ import androidx.appcompat.app.AppCompatActivity
 import com.example.testapp.databinding.ActivityMainBinding
 import com.google.android.material.snackbar.Snackbar
 
-const val COMPILE_TIME_CONSTANT: String = "Mike change 2"
+const val COMPILE_TIME_CONSTANT: String = "Mike change 2" ///CHANGE
```

Comment:
Change line to "Matt PR change based on comments

Reply:
Completed.